### ISSUE TITLE
NVDA does not read message from showInformationMessage()

### ISSUE THREAD

**Issue by IlyaBiryukov** [Microsoft/vscode#55190]
Issue Type: <b>Bug</b>

Extension uses `vscode.window.showInformationMessage` to show a notification. User tries to use NVDA to read it.

Expected: NVDA can read the notification.

Actual: NVDA cannot read the notification.

VS Code version: Code 1.25.1 (1dfc5e557209371715f655691b1235b6b26a06be, 2018-07-11T15:43:53.668Z)
OS version: Windows_NT x64 10.0.16299

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-6700 CPU @ 3.40GHz (8 x 3408)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: disabled_software<br>rasterization: disabled_software<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Memory (System)|31.92GB (23.39GB free)|
|Process Argv|C:\Program Files\Microsoft VS Code\Code.exe|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (2)</summary>

Extension|Author (truncated)|Version
---|---|---
cpptools|ms-|0.17.7
vsliveshare|ms-|0.3.472

</details>

**Comment by bpasero** [comment]
@IlyaBiryukov steps to reproduce please, every notification in VSCode is alerted to NVDA, extensions make no difference.

**Comment by IlyaBiryukov** [comment]
The bug originally came in the accessibility test pass of Live Share.
https://devdiv.visualstudio.com/DevDiv/_workitems/edit/646335

**Comment by bpasero** [comment]
@IlyaBiryukov I cannot reproduce when I try with a simple "Hello World" notification:

![43879863-415599bc-9ba6-11e8-92d6-a83b2d2fe8e8](https://user-images.githubusercontent.com/900690/43879918-84e620f2-9ba6-11e8-87cb-2b5777fafa16.png)

At this point, please try to produce a minimal reproducible sample for me to test.

**Comment by cait1inhennessy** [comment]
@bpasero Can we please reopen this?  Here are the steps to reproduce:

1. Download Live Share extension for VSCode
2. Go to the "share" icon in the status bar, click it, and sign in to start a collaboration session
3. Click the name icon in the status bar
![capture4](https://user-images.githubusercontent.com/16658549/45174888-d29ad600-b1c0-11e8-8c28-b5ed47a16182.PNG)
4. Select "share server" in the quick pick menu that appears
5. Type any number in "Enter port to expose..."
6. A notification will appear saying "Exposed local TCP port 3000...".  This notification is not announced by the screen reader.

Note that this doesn't happen for all notifications, just this one that appears following the input box.

**Comment by bpasero** [comment]
@cait1inhennessy I cannot reproduce with the steps provided. Using NVDA speech viewer I get this:

![screen shot 2018-09-09 at 10 42 15](https://user-images.githubusercontent.com/900690/45262750-4b966980-b41d-11e8-82c0-dd1dfca0783c.png)

So there must be some configuration issue on where this was tested.

**Comment by cait1inhennessy** [comment]
@bpasero Thanks!
